handbook/communication/content_guidelines/voice_and_tone.md

```diff
@@ -0,0 +1,95 @@
+# Voice and tone
+
+One of the ways we create a consistent and familiar experience for our users is by speaking with a consistent voice throughout our product and content while being aware of our tone.
+
+## What’s the difference between voice and tone?
+
+What’s the difference between voice and tone? Think of it this way: our **voice** is a personification of who we are. We always have the same voice. But, our **tone** changes to match our context and situation, usually through word choice.
+
+## Voice guidelines
+
+Our brand has a distinct voice and character. This voice and character is the basis of our writing for all of our digital platforms.
+
+When we write in our shared voice, we are always:
+
+### (This section is todo)
+
+- (This section is todo)
+    
+## Using tone
+
+Our tone is more than just the words we choose: it’s the way that we choose to share our personality through our voice.
```

Comment:
We should decide what is Sourcegraph's tone and write guidelines in this section @quinnkeast

Reply:
Based on a quick session with @kghopson, I've added some basic voice guidelines following our existing guidelines. The next step, once we have defined brand character attributes that act as the foundation of our brand personality, will be to update these voice guidelines based on those attributes.